.circleci/config.yml

```diff
@@ -98,6 +98,22 @@ jobs:
           command: |
             source py3-lint-env/bin/activate
             black --check --diff cla_public
+  webhint:
+    docker:
+      - image: circleci/python:latest-node-browsers
```

Comment:
I would start with the node image. We don't need to actually run our Python code so I don't think we need the Python image, and using the node one gets us ready to install hint much more easily. Unless we can't stop it installing from our project's package.json, but I think we should be able